.gitignore

```diff
@@ -1,5 +1,4 @@
 # Ignore  My Mac system files
 .DS_store
-
 # Ignore test.md
-./test.md 
\ No newline at end of file
+test.md 
```

Comment:
- [ ] **Required** There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up linters).Please add `node_modules` also.

Reply:
Addressed ✔